cloud-inquisitor/instrumentation/newrelic/context.go

```diff
@@ -0,0 +1,24 @@
+package newrelic
+
+import (
+	"context"
+
+	"github.com/RiotGames/cloud-inquisitor/cloud-inquisitor/logger"
+	"github.com/newrelic/go-agent/v3/newrelic"
+	"github.com/sirupsen/logrus"
+)
+
+func GetTxnFromLambdaContext(ctx context.Context, logger *logger.Logger) *newrelic.Transaction {
+	if txn := newrelic.FromContext(ctx); txn != nil {
+		logger.WithFields(logrus.Fields{}).Debug("generated new relic transaction from lambda context")
```

Comment:
After all this is OSS, Shall we give another logging option rather than newrelic?

Reply:
I would assume we would eventually support another method. We just only currently have the need for New Relic so that integrations will stop there. Although I do think it would be easy to add another layer of abstraction to make this possible.